pypasta/LinuxMailCharacteristics.py

```diff
@@ -221,48 +220,35 @@ class LinuxMailCharacteristics:
 
     def _is_from_bot(self, message):
         email = self.mail_from[1].lower()
-        bots = ['broonie@kernel.org', 'lkp@intel.com']
-        potential_bot = True in [bot in email for bot in bots]
-
-        if potential_bot and \
-           email_get_header_normalised(message, 'x-patchwork-hint') == 'ignore':
-            return True
-
+        bots = ['tip-bot2@linutronix.de', 'tipbot@zytor.com', 'noreply@ciplatform.org', 'syzbot',
+                'syzkaller.appspotmail.com']
+        potential_bots = ['broonie@kernel.org', 'lkp@intel.com']
         subject = email_get_header_normalised(message, 'subject')
+        uagent = email_get_header_normalised(message, 'user-agent')
+        xmailer = email_get_header_normalised(message, 'x-mailer')
 
+        if email in bots:
```

Comment:
This check is incorrect for 'syzbot', as it's not a valid email address. However, we to classify all mails with 'syzbot' in their mail adress as bots. Will fix this.

Reply:
Ok, thanks